ci/docker-run.sh

```diff
@@ -72,19 +72,25 @@ ARGS+=(
   --env CI_JOB_ID
   --env CI_PULL_REQUEST
   --env CI_REPO_SLUG
-  --env CODECOV_TOKEN
   --env CRATES_IO_TOKEN
 )
 
+# Also propagate environment variables needed for codecov
+# https://docs.codecov.io/docs/testing-with-docker#section-codecov-inside-docker
+# We normalize CI to `1`; but codecov expects it to be `true` to detect Buildkite...
+CODECOV_ENVS=$(CI=true bash <(curl -s https://codecov.io/env))
```

Comment:
Because of https://github.com/koalaman/shellcheck/wiki/SC2207, [adding to `ARGS`](6630dd95368c0e3b28afb5c44bbd68d8dab54d32) didn't work well.